app/assets/javascripts/controllers/DragDropCtrl.coffee

```diff
@@ -0,0 +1,85 @@
+# TODO: create modal for save option?
+@bruseApp.controller 'DragDropCtrl', ['$scope', '$http', ($scope, $http) ->
+    $scope.fileObject = null
+    $scope.dataObject = {location:''}
+    $scope.imageFile = null
+    $scope.isDropped = false
+    $scope.isSaved = false
+    $scope.file = []
+
+    $scope.saveOpt = (location) ->
+      $scope.isDropped = false
+      $scope.dataObject.location = location
+
+      if location != ''
+        # send object to server
+        $http.post('/upload_drop.json', $scope.dataObject).then((response) ->
+          $scope.file = response.data.file;
+          if $scope.file.id != null
+            $scope.isSaved = true
+          else
+            $scope.file = []
+            $scope.isSaved = false
+          )
+        .catch((response) ->
+          console.error "Couldn't save.."
+          )
+]
+
+@bruseApp.directive('bDropzone', () ->
```

Comment:
Yes, since the directive is placed inside the controller scope as can be seen [here](https://github.com/tkomstadius/bruse/pull/36/files#diff-8dc8b47ebcec2d23b676ca3cc9e9582aR8).